docs/articles/deployments.md

```diff
@@ -0,0 +1,237 @@
+# Deployments
+
+# What is a deployment
+Every time a user creates a new [service](services.md), modifies or removes an existing one, new deployment is created. Basically, the deployment is a series of steps that Marathon has to execute in order to fulfil the user request. Let's consider an example where a few services are already deployed and running:
+```
+/product
+   /database
+     /mysql
+     /mongo
+   /service
+     /play-app
+```
+Now, a new service `/product/service/rails-app` needs to be deployed. To achieve this the user submits the service definition to the `/v2/apps` [Rest API](api.md) endpoint:
+```
+{
+  "id": "/product/service/rails-app",
+  "cpus": 0.3,
+  "instances": 1,
+  "mem": 2048,
+  "cmd": ...
+}
+```
+If the provided definition is valid and there are no [deployment conflicts](deployments.md#forced-deployments) Marathon will return a `HTTP 201 (created)` response with a normalized service definition (including default fields that are allowed to be omitted) with an extra `deployments` field containing the deployment Id:
```

Comment:
If I'm not mistaken we also set the deployment id in an extra header. Not sure if this is worth mentioning.

Reply:
Good point! I'll add a sentence about it.